src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.js

```diff
@@ -169,10 +169,13 @@
     $(".page").hide();
     $("#stackpage").show();
 
-    $("#rawExceptionButton").click(function () {
+    $("#rawExceptionButton").click(function (event) {
```

Comment:
Ok, looking at the rest of the codes, we're not using jquery. We're would be the right thing.